### PR TITLE
minify leaflet scripts using uglifier

### LINKED ISSUE #8158: Minify leaflet scripts using Uglifier
**Please describe the desired behavior.**
Currently Uglifier is only minifying the scripts defined in assets but not in /lib. So we need to minify leaflet scripts as well to bring down the initial load time.

**Additional context (optional)**

image

### PR DESCRIPTION
**Pull request by Tlazypanda**
Fixes #8158 

Minified leaflet scripts using Uglifier.

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

## Screenshots  

Minify js suggestion not present anymore   

after_mi

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu can you kindly review? Thanks :v:

File: config/environments/production.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@
   # Compress JavaScripts and CSS
   config.assets.compress = true
   config.assets.js_compressor = Uglifier.new(harmony: true)
+  Uglifier.new(harmony: true).compile(File.read("public/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.js"))
+  Uglifier.new(harmony: true).compile(File.read("public/lib/leaflet-environmental-layers/lib/glify.js"))
```
Comment (jywarren) [inline config/environments/production.rb:20]: Will these then be grabbed automatically by asset pipeline? Awesome, thank you @Tlazypanda you're really an expert in all this!!!

**Comment by Tlazypanda** [comment]
@jywarren on checking the performance the Uglifier scripts to be minified are not getting minified https://github.com/publiclab/plots2/pull/8159  but locally they are so maybe fetching from cache can we do a manual rake assets:precompile (possibly after removing the assets folder once) and then letting it build a fresh assets folder :sweat_smile:

**Comment by jywarren** [comment]
Perhaps you could write this into a Rails migration, sit hat it automatically runs upon publishing to stable and production? What do you think? I'm happy to help with that if you want to open a PR !

**Pull request merged**